src/main/java/core/basesyntax/service/impl/AddOperationHandlerImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationHandler;
+
+public class AddOperationHandlerImpl implements OperationHandler {
+    private final FruitDao fruitDao;
+
+    public AddOperationHandlerImpl(FruitDao fruitDao) {
+        this.fruitDao = fruitDao;
+    }
+
+    @Override
+    public void operationWithFruitTransaction(FruitTransaction fruitTransaction) {
+        fruitDao.getAll().merge(
```

Comment:
Let's use dao methods to save updated data

Reply:
I'm already using the getAll method from Dao, and the merge method puts the new value under the given key (if none) or updates the existing key with the given value, but for this situation i think it's better to use merge then put.